openapi/v1/integrationCurator.yaml

```diff
@@ -3503,6 +3503,189 @@ paths:
       summary: Retrieve study metadata by querying sample ID (accession)
       tags:
       - Study integration as Curator
+  /api/v1/as-curator/integration/link/studies/by/files:
+    get:
+      description: |+
+        Retrieve study metadata objects whose linked attached files metadata matches all supplied conditions.
+
+        ## Conditions
+        It is possible to supply conditions for:
+
+        1. Attachment files (full-text or metadata key-value pair)
+        2. Parent studies (full-text or metadata key-value pair)
+
+        ## Metadata full-text queries
+        Single words can be supplied as is; otherwise, use speech marks (`"`) to quote queries that include whitespace. Speech marks and backslash characters in the query need to be escaped with a backslash (`\`).
+
+        ## Metadata filters
+        Metadata filters are key-value pairs joined by an operator. The `=` operator matches literal values/string. The `!=` operator matches anything except the literal value/string. The `<` or `>` operators match numerical results that are less or greater than the supplied value. Strings containing whitespace need to be quoted with (`"`).
+
+        ## Data queries
+        Data queries must be a list of key-value pairs, separated by whitespace. The set of valid keys depends on the specific query type, and is documented in the query parameter summary. The values can be simple non-whitespace strings, or strings enclosed by speech marks (`"`). Depending on the key, the value may be be a comma-delimited list of string values. Others require numerical values. Ranges can be supplied with `(0:1)` syntax, OR values with `|`.
+
+        ## Combinations
+        Metadata queries/filters for the same parameter can be combined with `&&`, `AND`, `||` and `OR` operators, using white-space to separate out the terms and operators. Parentheses `( )` can be used for complex expressions.
+
+        ## Versioning
+        Specific versions of omics data files (eg. GCT) can be queried via the useVersions parameter. Different versions of an omics data file are associated via their CHAIN_ID metadata value. This CHAIN_ID can be supplied to the useVersions parameter along with the version number or specific omics data file accessions to include them in the query. If nothing is supplied to the useVersions parameter then only the active version (which is usually the last one imported) is queried. This acts as a filter before the rest of the query is carried out.
```

Comment:
I understand that this is minor, but it seems that chaining of signal data is not applicable to studies

Reply:
thanks for the catch! Removed